_patternlab-files/partials/patternNav.twig

```diff
@@ -0,0 +1,60 @@
+<ul class="patterns-menu sg-nav">
+    <li class="sg-nav-"><a class="sg-acc-handle">base</a>
+        <ul class="sg-acc-panel">
+
+            <li class="sg-nav-"><a class="sg-pop" href="app/bower_components/pattern-library/patterns/base/blockquote">Blockquote</a></li>
```

Comment:
These shouldn't be pointing to the bower_components, they should be pointing to the compiled html files